pkgs/development/python-modules/annotated-types/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, hatchling
+, typing-extensions
+}:
+
+buildPythonPackage rec {
+  pname = "annotated-types";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "annotated-types";
+    repo = "annotated-types";
+    rev = "v${version}";
+    hash = "sha256-zCsWfJ8BQuov8FN+hlm9XBKWAAQ/KHPK/x024A8k2kE=";
+  };
+
+  nativeBuildInputs = [
+    hatchling
+  ];
+
+  propagatedBuildInputs = [
+    typing-extensions
+  ];
+
```

Comment:
Would you please add `pytestCheckHook`?

Reply:
Beautiful:
![image](https://github.com/NixOS/nixpkgs/assets/7548295/ea34ef20-bd83-466e-9482-7292e265ac30)